pandas/core/generic.py

```diff
@@ -99,6 +99,32 @@ def _single_replace(self, to_replace, method, inplace, limit):
     return result
 
 
+def _update_column(this, that, overwrite=True, filter_func=None,
+                   raise_conflict=False):
```

Comment:
I think we have an issue, but pls create one to deprecate this arg and rename to ``errors=`` (you can do that in the internal function and put a TODO in the external facing ones).